pkg/shell-operator/bootstrap.go

```diff
@@ -85,7 +84,7 @@ func RequireExistingDirectory(inDir string) (dir string, err error) {
 	return dir, nil
 }
 
-func EnsureTempDirectory(inDir string) (string, error) {
+func ensureTempDirectory(inDir string) (string, error) {
```

Comment:
This one is also use in addon-operator
https://github.com/flant/addon-operator/blob/e4887ca4cd1b248ea7ec6d6f326d96cf77ea9a32/pkg/addon-operator/bootstrap.go#L34